packages/material-ui-unstyled/src/ModalUnstyled/ModalManager.test.js

```diff
@@ -166,6 +166,62 @@ describe('ModalManager', () => {
       expect(container1.style.paddingRight).to.equal('20px');
       expect(fixedNode.style.paddingRight).to.equal('');
     });
+
+    it('should restore styles correctly if overflow existed before', () => {
+      const modal = {};
+      const container2 = document.createElement('div');
+      container2.style.overflow = 'overlay';
+
+      Object.defineProperty(container2, 'scrollHeight', {
+        value: 100,
+        writable: false,
+      });
+      Object.defineProperty(container2, 'clientHeight', {
+        value: 100,
+        writable: false,
+      });
+
+      document.body.appendChild(container2);
+      modalManager.add(modal, container2);
+      modalManager.mount(modal, {});
+
+      expect(container2.style.overflow).to.equal('hidden');
+      modalManager.remove(modal);
+
+      expect(container2.style.overflow).to.equal('overlay');
+      expect(fixedNode.style.paddingRight).to.equal('');
+
+      document.body.removeChild(container2);
```

Comment:
`container2` will not be cleaned up properly if the test fails i.e. this test breaking could lead to "catastrophic" failure of the whole test suite.

We can move the test into a separate describe block where `beforeEach/afterEach` setup/clean up this dedicated container.

Reply:
Sure, I'll do that